src/sentry/eventstore/snuba/backend.py

```diff
@@ -220,7 +220,7 @@ def get_event_by_id(self, project_id, event_id, group_id=None):
             try:
                 result = snuba.raw_query(
                     dataset=dataset,
-                    selected_columns=["group_id"],
+                    selected_columns=self.__get_columns(dataset),
```

Comment:
This makes sure we return the occurrence_id from snuba here when using the issue platform dataset. It results in a few extra columns for other folks that are using this function, but considering it's a single row this shouldn't be a big deal